optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -14,6 +14,10 @@
 
   <name>Employee Rostering as a Service GWT UI</name>
 
+  <properties>
+    <version.org.webjars.bower.org.patternfly>3.18.1</version.org.webjars.bower.org.patternfly>
```

Comment:
all versions must be declared in the parent pom

Reply:
Oops. Moved.